oak-run-commons/src/test/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedTreeStoreIT.java

```diff
@@ -675,6 +675,7 @@ private PipelinedTreeStoreStrategy createStrategy(MongoTestBackend backend, Pred
                 pathPredicate,
```

Comment:
The tests miss the case of modified date which I get is difficult to test. But can we add UT/ITs for cases like removeNode and the procedure using updateIndexStore above?

Reply:
Yes I'll try to add more tests!